phpBB/phpbb/install/module_base.php

```diff
@@ -110,10 +110,38 @@ public function run()
 
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+		    // Receiving task index
```

Comment:
I think this is the example https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/install/module_base.php#L123
Initially, I used task index as you said. However, the user could skip over a step without noticing it. Thus, I had to return him the data about which tasks are already completed.
``` $iterator->current()->get_navigation_stage_path() ```

Ok, I will try to retrieve the previously submitted infomation.